atomisticparsers/gromacs/parser.py

```diff
@@ -1629,6 +1678,53 @@ def parse_input(self):
         )
         sec_control_parameters.m_set(quantity_def, input_parameters)
 
+    def find_trajectory_files(self):
+        """
+        Find and set trajectory files following the priority:
+        "trr" > "xtc", fallback "pdb" > "gro". Prioritize files with exact basename
+        match over files with partial basename match with the same extension.
+        Test other files only if no true trajectory format with exact or partial
+        basename match is found.
+
+        Example: out.log -> out.trr > out_1.trr > 2_out.trr > out.xtc > ... > out.pdb > out.gro
```

Comment:
Example - For out.log: out.trr > out_1.trr

Just the -> is too close to >

Add the additional sorting explanation if added to type 1 above